<launch>

  <node pkg="demo_rgbd" type="featureTracking" name="featureTracking" output="screen">
    <remap from="/camera/rgb/image_rect" to="/image/raw"/>
  </node>
  <node pkg="demo_rgbd" type="visualOdometry" name="visualOdometry" output="screen">
    <remap from="/depth_cloud" to="/sync_scan_cloud_filtered"/>
  </node>
  <node pkg="demo_rgbd" type="bundleAdjust" name="bundleAdjust" output="screen"/>
  <node pkg="demo_rgbd" type="processDepthmap" name="processDepthmap" output="screen"/>
  <node pkg="demo_rgbd" type="stackDepthPoint" name="stackDepthPoint" output="screen"/>
  <node pkg="demo_rgbd" type="transformMaintenance" name="transformMaintenance" output="screen"/>
  <node pkg="demo_rgbd" type="registerPointCloud" name="registerPointCloud" output="screen"/>

  <!--node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find demo_rgbd)/demo_rgbd.vcg"/-->

</launch>
